INTRODUCTION.md

```diff
@@ -1387,6 +1420,27 @@ The purged messages in queue will fail with error code set to
 `RD_KAFKA_RESP_ERR__PURGE_QUEUE`.
 
 
+#### Fatal consumer errors
+
+A consumer configured for static group membership (`group.instance.id`) may
+raise a fatal error if a new consumer instance is started with the same
```

Comment:
clearer if you change 'a new' to 'another'

Reply:
I think new is more clear, since that is what it technically is, a newer instance of the app.
 keeping it.

Reply:
Wouldn't the new consumer be the one that's fenced rather than the existing one?

> The edge case is that if we could have members with the same `group.instance.id` (for example mis-configured instances with a valid `member.id` but added a used `group.instance.id` on runtime). When `group.instance.id` has duplicates, we could refuse join request from members with an outdated `member.id`, since we update the mapping upon each join group request. In an edge case where the client hits this exception in the response, it is suggesting that some other consumer takes its spot. The client should immediately fail itself to inform end user that there is a configuration bug which is generating duplicate consumers with same identity.

Reply:
The way I read that is the old consumer is fenced off, which is what makes sense, you want to terminate old stale instances when new are created.

Reply:
Yeah you are right.